index.html

```diff
@@ -9,39 +9,290 @@
+                    <li><a href="microverse.org" class="item"><img src="img/svg/Vector-5.svg" /></a></li>
+                    <li><a href="microverse.org"><img src="img/svg/Vector-6.svg" /></a></li>
+                    <li><a href="microverse.org" class="item"><img src="img/svg/Vector-7.svg" /></a></li>
+                    <li><a href="microverse.org" class="item"><img src="img/svg/Vector-8.svg" /></a></li>
+                    <li><a href="microverse.org" class="item"><img src="img/svg/Vector-9.svg" /></a></li>
+                    <li><a href="microverse.org" class="item"><img src="img/svg/Vector-10.svg" /></a></li>
+                </ul>
+
+                <div>
+                    <div class="action">
+                        <button class="buttonaction">Get my resume</button>
+                    </div>
+                </div>
+                <ul class="languager">
+                    <li>
+                        <div class="flex">
+                            <h3>Languages</h3>
+                            <img src="img/Union.png" alt="">
+                        </div>
+                        <ul class="dropdown">
+                            <li>
+                                <img src="img/Ellipse 1.png" alt="">
+                                <p>javascript</p>
+                            </li>
+                            <li>
+                                <img src="img/Ellipse 1 (1).png" alt="">
+                                <p>html</p>
+                            </li>
+                            <li>
+                                <img src="img/Ellipse 1 (2).png" alt="">
+                                <p>css</p>
+                            </li>
+                        </ul>
+                    </li>
+                    <li>
+                        <div class="flex">
+                            <h3>Framework</h3>
+                            <img src="img/Union.png" alt="">
+                        </div>
+                    </li>
+                    <li>
+                        <div class="flex">
+                            <h3>Skills</h3>
+                            <img src="img/Union.png" alt="">
+                        </div>
+                    </li>
+                </ul>
+
+
+            </main>
+
+        </section>
+
+        <section class="contactsession">
+            <h1>Contact me</h1>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project
+                that
+                needs coding. I’d love to help with it </p>
+
+            <form action="#" method="post">
+                <input type="text" placeholder="Enter name" maxlength="30">
+                <input type="email" placeholder="Enter email">
+                <textarea name="" id="" placeholder="Write your message here"></textarea>
+                <button type="submit">Get In Touch</button>
+            </form>
+        </section>
```

Comment:
- [ ] Please remove the contact section from inside of your about myself section, as it is a section in itself and not part of the about myself section, You can even remove it for now since it is not part of this project and you will ask for code review for this project in the next milestone, The `<section>`  element represents a generic standalone section of a document